webapi/src/test/scala/org/knora/webapi/util/DateUtilSpec.scala

```diff
@@ -52,6 +54,25 @@ class DateUtilSpec extends WordSpec with Matchers {
             reverseConvertedBundesbriefDateValueV1 should be(bundesbriefDateValueV1)
         }
 
+        "convert an era date string with year precision to a Java GregorianCalendar BC" in {
+            val dateRange: DateRange = DateUtilV1.dateString2DateRange("50-02-28 BC", KnoraCalendarV1.GREGORIAN)
+            dateRange.start.get(Calendar.ERA) should be(GregorianCalendar.BC)
```

Comment:
Could you add a test involving a BC date and explicitly checking for the Julian Day Numbers?